nixos/modules/services/networking/mycelium.nix

```diff
@@ -0,0 +1,119 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.mycelium;
+in
+{
+  options.services.mycelium = {
+    enable = lib.mkEnableOption "mycelium network";
+    peers = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      description = "List of peers to connect to";
```

Comment:
This needs a longer description / defaultText, to make it obvious the hosted peers are also added, at least it it's not explicitly disabled.

Reply:
wouldn't that be better in the option where we add the hosted peers?

Reply:
Just by looking at the "peers" config option, this strongly suggests there's no other peers.

The boostrap peers are not in that list, they're not discoverable via `config.` either.

What about my initial proposal, calling this `extraPeers`, and documenting it with "List of peers to connect to, in addition to these potentially enabled by `addHostedPublicNodes`"?

That should make it clear that these are not the only peers.

Reply:
There's been a push at around the time I wrote this comment, now adding documentation, but keeping it `peers`, not `extraPeers`. Im not sure you're done. If so, please write a conclusion and mark the thread as resolved.

Reply:
I still think peers is clearer. since extraPeers seems like peers but with more noise?